src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs

```diff
@@ -279,11 +306,19 @@ void UpdateBadgeViewPlacement(bool force = false)
                 contentMargin = margins.Item2;
             }
 
+            BadgeText.Margin = GetMarginsLengthIsOne();
             BadgeIndicatorContainer.Margin = containerMargin;
             BadgeContent.Margin = contentMargin;
             placementDone = true;
         }
 
+        private Thickness GetMarginsLengthIsOne()
+			=> BadgeText.Text.Length == 1 && IsRounded
+				? Device.RuntimePlatform == Device.Android  
+					? new Thickness(10,0,10,0)
+					: new Thickness(8,0,8,0)
+				: new Thickness(4,0,4,0);
```

Comment:
isn't more readable with a nested if else statement instead ?

Reply:
About these `magic numbers`, will they work in all screen resolutions/sizes?

Reply:
@pictos Magic numbers working well, testing in several resolutions, sizes and badgeview seems corectly